### ISSUE TITLE
`<meta charset="utf-8">` in Astro source is detected as a wrong error

### ISSUE THREAD

**Issue by SaiYs** [markuplint/markuplint#803]
- Markuplint version: `3.6.1`
- Parser lang: Astro (@markuplint/astro-parser@3.6.1)
- Node.js version: `19.8.1`
- OS: macOS

## Describe the bug

markuplint emits false errors for Astro Layout file.

## Code Example

This code passed successfully.

```astro
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Title</title>
    <meta charset="utf-8" />
  </head>
  <body> <slot /></body>
</html>
```

However, this will emits errors.

```astro
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Title</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width" />
    <meta name="generator" content={Astro.generator} />
    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
  </head>
  <body> <slot /></body>
</html>
```

If and only if `<head>` has only `<title>` and `<meta charset="utf-8" />`, it passes.
Below code fails too.

```astro
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Title</title>
    <meta charset="utf-8" />
    <!-- <meta name="viewport" content="width=device-width" /> -->
    <!-- <meta name="generator" content={Astro.generator} /> -->
    <!-- <link rel="icon" type="image/svg+xml" href="/favicon.svg" /> -->
  </head>
  <body> <slot /></body>
</html>
```

## Steps To Reproduce

Above.

## The current behavior

```sh
> pnpm markuplint src/layouts/min.astro

<markuplint> error: The "charset" attribute expects utf-8 (invalid-attr) /path/to/min.astro
   4: ••••<title>Title</title>
   5: ••••<meta•charset="utf-8" />
    <meta name="viewport" content="width=device-width" />
    <meta name="generator" content={astro.generator} />
    <link rel="icon" type="image/svg+xml" href="/favicon.svg
   6: ••••<meta•name="viewport"•content="width=device-width"•/>
<markuplint> error: Require the "meta[charset="UTF-8" i]" element (required-element) /path/to/min.astro
   2: <html•lang="en">
   3: ••<head>
   4: ••••<title>Title</title>
```

## The expected behavior

should be passed.

**Comment by YusukeHirao** [comment]
@SaiYs 

I was able to reproduce the problem.
Thanks for reporting this – I'll address the issue.

**Comment by SaiYs** [comment]
Thanks to speedy fix!
